quickstarts/jfrog-platform/config.yml

```diff
@@ -0,0 +1,62 @@
+# quickstarts/example-category/example-quickstart/config.yml
+
+# This is automatically created by the system, this field should not be included by the author when creating a new quickstart.
+id:
```

Comment:
We can remove these lines, the system will automatically generate it for us!
```suggestion
```

Reply:
Got it, removing id